### ISSUE TITLE
Default network choice ... wonky?

### ISSUE THREAD

**Issue by dwt** [nschloe/tiptop#81]
The default network choice to display seems wonky for me, my main interface (wlan) is en0 (Mac user here), but instead en5 is chosen for display, which has no ipv4 and only a very small part of the traffic going through it.

Not quite sure why that is and what the best way to choose a default interface with `psutil` would be, but right now this is a bit strange.

Also good enough as a workaround would be the ability to change the network interface from inside the app without having to restart it (and loose history that way).

**Comment by nschloe** [comment]
Yeah, tiptop isn't really made for dynamically choosing a network interface yet. I improved autoselection a bit (https://github.com/nschloe/tiptop/pull/83) so you get en0 by default. If that doesn't work, you can always use the `--net` argument on the command line.

**Comment by dwt** [comment]
Well, this sorta works. I now get the widget awdl0 chosen by default. I still think the real fix would be to choose the interface with most traffic as default. I guess that is something to look into how `btop` and others do this.

**Comment by nschloe** [comment]
Sure! Pull requests welcome.
